### PR TITLE
Use different port H2 DB datasource URL

### LINKED ISSUE #334: Admin local will not start if Kafka is already running locally
The default H2 port is 9092, same as the default port for Kafka brokers. Unless there is a very specific need to to use port 9092 for H2, I would suggest to customize that value for our embedded DB.

### PR DESCRIPTION
**Pull request by ilayaperumalg**
- 9092 collides with Kafka default port

Resolves #334

### CODE REVIEW

File: spring-cloud-dataflow-admin-local/src/main/resources/admin.yml [hunk 1]
```diff
@@ -18,7 +18,7 @@ spring:
     config:
       uri: http://localhost:8888
   datasource:
-    url: jdbc:h2:tcp://localhost:9092/mem:dataflow
+    url: jdbc:h2:tcp://localhost:9093/mem:dataflow
```
Comment (markfisher) [inline spring-cloud-dataflow-admin-local/src/main/resources/admin.yml]: This is still in the range that would be commonly used for a Kafka cluster. Maybe `19092`?

Comment (ilayaperumalg) [inline spring-cloud-dataflow-admin-local/src/main/resources/admin.yml]: oh yeah, sure.

**Comment by ilayaperumalg** [comment]
updated the PR.

**Comment by markfisher** [comment]
LGTM; squashed and merged as https://github.com/spring-cloud/spring-cloud-dataflow/commit/6a07dd99f722f4b5faa2ee835c9f973f129d92b0

**Pull request closed without merging**